src/bp-core/admin/bp-core-admin-tools.php

```diff
@@ -871,6 +871,246 @@ function bp_core_admin_debug_information( $debug_info = array() ) {
+				'debug' => defined( 'BP_USE_WP_ADMIN_BAR' ) ? BP_USE_WP_ADMIN_BAR : 'undefined',
+			),
+			'BP_ENABLE_USERNAME_COMPATIBILITY_MODE' => array(
+				'label' => 'BP_ENABLE_USERNAME_COMPATIBILITY_MODE',
+				'value' => defined( 'BP_ENABLE_USERNAME_COMPATIBILITY_MODE' ) ? BP_ENABLE_USERNAME_COMPATIBILITY_MODE : __( 'Undefined', 'buddypress' ),
+				'debug' => defined( 'BP_ENABLE_USERNAME_COMPATIBILITY_MODE' ) ? BP_ENABLE_USERNAME_COMPATIBILITY_MODE : 'undefined',
+			),
+			'BP_AVATAR_DEFAULT_THUMB' => array(
+				'label' => 'BP_AVATAR_DEFAULT_THUMB',
+				'value' => defined( 'BP_AVATAR_DEFAULT_THUMB' ) ? BP_AVATAR_DEFAULT_THUMB : __( 'Undefined', 'buddypress' ),
+				'debug' => defined( 'BP_AVATAR_DEFAULT_THUMB' ) ? BP_AVATAR_DEFAULT_THUMB : 'undefined',
+			),
+			'BP_AVATAR_DEFAULT' => array(
+				'label' => 'BP_AVATAR_DEFAULT',
+				'value' => defined( 'BP_AVATAR_DEFAULT' ) ? BP_AVATAR_DEFAULT : __( 'Undefined', 'buddypress' ),
+				'debug' => defined( 'BP_AVATAR_DEFAULT' ) ? BP_AVATAR_DEFAULT : 'undefined',
+			),
+			'BP_AVATAR_URL' => array(
+				'label' => 'BP_AVATAR_URL',
+				'value' => defined( 'BP_AVATAR_URL' ) ? BP_AVATAR_URL : __( 'Undefined', 'buddypress' ),
+				'debug' => defined( 'BP_AVATAR_URL' ) ? BP_AVATAR_URL : 'undefined',
+			),
+			'BP_AVATAR_UPLOAD_PATH' => array(
+				'label' => 'BP_AVATAR_UPLOAD_PATH',
+				'value' => defined( 'BP_AVATAR_UPLOAD_PATH' ) ? BP_AVATAR_UPLOAD_PATH : __( 'Undefined', 'buddypress' ),
+				'debug' => defined( 'BP_AVATAR_UPLOAD_PATH' ) ? BP_AVATAR_UPLOAD_PATH : 'undefined',
+			),
+			'BP_SHOW_AVATARS' => array(
+				'label' => 'BP_SHOW_AVATARS',
+				'value' => defined( 'BP_SHOW_AVATARS' ) && (bool) BP_SHOW_AVATARS ? __( 'Enabled', 'buddypress' ) : __( 'Disabled', 'buddypress' ),
+				'debug' => defined( 'BP_SHOW_AVATARS' ) ? BP_SHOW_AVATARS : 'undefined',
+			),
+			'BP_AVATAR_ORIGINAL_MAX_FILESIZE' => array(
+				'label' => 'BP_AVATAR_ORIGINAL_MAX_FILESIZE',
+				'value' => BP_AVATAR_ORIGINAL_MAX_FILESIZE,
+			),
+			'BP_AVATAR_ORIGINAL_MAX_WIDTH' => array(
+				'label' => 'BP_AVATAR_ORIGINAL_MAX_WIDTH',
+				'value' => BP_AVATAR_ORIGINAL_MAX_WIDTH,
+			),
+			'BP_AVATAR_ORIGINAL_MAX_FILESIZE' => array(
+				'label' => 'BP_AVATAR_ORIGINAL_MAX_FILESIZE',
+				'value' => size_format( BP_AVATAR_ORIGINAL_MAX_FILESIZE ),
+				'debug' => BP_AVATAR_ORIGINAL_MAX_FILESIZE,
+			),
+			'BP_AVATAR_FULL_HEIGHT' => array(
+				'label' => 'BP_AVATAR_FULL_HEIGHT',
+				'value' => BP_AVATAR_FULL_HEIGHT,
+			),
+			'BP_AVATAR_FULL_WIDTH' => array(
+				'label' => 'BP_AVATAR_FULL_WIDTH',
+				'value' => BP_AVATAR_FULL_WIDTH,
+			),
+			'BP_AVATAR_THUMB_HEIGHT' => array(
+				'label' => 'BP_AVATAR_THUMB_HEIGHT',
+				'value' => BP_AVATAR_THUMB_HEIGHT,
+			),
+			'BP_AVATAR_THUMB_WIDTH' => array(
+				'label' => 'BP_AVATAR_THUMB_WIDTH',
+				'value' => BP_AVATAR_THUMB_WIDTH,
+			),
+			'BP_TESTS_DIR' => array(
```

Comment:
I would expect this never being set in a live community site. Should we remove it? 🤔

Reply:
Yes 👍